.gitignore

```diff
@@ -1,7 +1,8 @@
-tests/Config.php
+tests/config/config.php
```

Comment:
This change should be reflected in the `README.md` as it talks about how to prepare for running the tests.